cubedash/_pages.py

```diff
@@ -244,6 +244,24 @@ def about_page():
     return utils.render("about.html")
 
 
+@app.route("/dashboard")
+def dashboard_page():
+    dc = datacube.Datacube()
```

Comment:
This will create new connection pools on every page load, and ignore some user config. It's better to reuse the existing ODC index:
```suggestion
    dc = Datacube(index=_model.STORE.index)
```

Reply:
Thanks